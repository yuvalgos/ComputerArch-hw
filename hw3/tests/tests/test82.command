./dflow_calc tests/test82.dat tests/test82.in d87 p30 p174 p170 p75 d43 d132 d80 d95 p24 d195 d83 p132 d182 d181 d190 p99 d18 p59 p120 p151 p81 d133 d192 p58 p167 p7 p21 p100 d77 d24 p195 d74 p108 d160 p16 p1 d91 d5 p133 p3 p147 d22 d108 d182 p18 p126 p20 p20 p71 p13 p22 d34 p96 d197 p172 p41 p179 p180 p93 p57 d65 p76 d102 p56 d13 d118 p104 p77 p149 p122 p14 d22 p39 d79 d150 d140 p183 d36