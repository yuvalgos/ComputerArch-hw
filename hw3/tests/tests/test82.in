0 23 0 26
5 26 14 25
4 18 12 30
5 9 3 10
8 18 30 2
7 22 20 14
5 10 7 12
6 8 10 12
8 28 17 26
1 23 11 4
4 26 14 24
4 14 27 0
3 26 22 6
1 9 9 8
7 15 23 4
1 15 17 10
6 6 9 5
2 28 27 5
3 27 17 8
6 13 13 29
1 23 8 5
5 0 0 10
3 17 4 4
3 30 17 21
5 6 27 2
8 22 7 14
6 31 20 15
6 30 29 27
3 31 15 4
6 25 6 26
4 7 16 27
1 20 1 4
1 10 4 5
8 16 18 6
4 17 1 16
7 1 19 3
7 7 9 31
2 8 6 20
6 2 18 31
8 13 3 11
6 19 26 21
3 17 9 7
5 29 9 17
5 29 5 1
5 5 23 26
2 28 25 18
8 31 9 6
5 4 15 15
7 3 24 4
1 5 21 15
1 13 19 31
5 25 23 6
8 5 20 7
2 10 25 7
0 26 24 27
1 18 3 15
7 24 23 19
3 6 5 11
0 27 7 31
3 27 14 8
4 10 11 24
0 6 16 4
1 13 14 19
6 31 16 31
3 30 3 2
0 28 20 9
3 14 13 17
0 7 12 16
7 24 25 2
0 19 19 26
3 4 26 8
8 24 1 24
7 1 23 1
0 29 13 20
6 22 6 9
8 28 3 23
5 26 27 27
1 23 9 5
1 18 3 27
2 30 19 4
2 29 5 3
6 27 5 10
7 0 29 6
6 11 30 15
8 24 13 21
3 21 0 30
0 30 18 8
8 28 28 26
3 30 12 31
3 23 31 3
0 10 17 8
8 15 7 11
1 24 25 4
3 16 12 12
4 29 29 15
4 0 23 23
4 15 4 20
4 7 4 12
3 5 0 7
3 4 8 14
3 13 5 19
4 7 15 24
3 0 7 31
8 16 20 11
7 29 24 21
2 3 17 12
0 1 5 25
5 18 8 14
4 30 2 2
4 31 19 16
7 23 13 26
4 23 27 9
0 31 21 10
8 1 0 20
4 3 30 18
3 21 30 10
2 13 11 13
4 1 9 9
2 26 6 10
2 8 5 6
4 27 24 26
2 27 13 21
0 15 31 9
7 31 5 22
1 25 8 8
2 10 14 26
4 17 9 31
8 7 29 6
5 2 6 2
3 11 10 24
0 30 30 21
7 20 28 26
8 28 17 20
0 25 12 12
8 31 19 20
4 28 28 7
8 7 0 4
8 22 13 13
7 21 20 19
4 11 16 28
2 12 12 11
6 1 12 19
8 31 20 27
4 29 13 25
5 29 0 29
3 0 17 14
5 21 9 1
8 12 4 22
2 2 15 7
1 8 20 25
5 26 11 1
3 13 11 5
7 19 16 5
7 4 25 17
5 23 19 2
2 30 9 22
1 13 3 21
5 17 10 22
6 1 23 26
2 0 28 12
4 13 20 8
7 6 15 3
7 6 17 21
1 13 0 26
2 14 29 21
1 14 7 24
5 6 9 30
5 3 12 17
4 4 31 21
4 10 17 29
7 28 0 1
8 17 20 18
2 29 21 21
2 5 7 1
5 8 12 13
3 30 29 5
6 14 12 3
8 18 15 8
4 12 10 2
1 29 13 4
6 23 3 1
1 31 24 8
1 2 15 7
5 20 3 8
7 15 25 10
4 15 28 5
7 13 16 3
4 12 5 17
0 22 19 19
1 3 24 18
5 0 28 20
1 15 13 14
5 16 19 28
3 6 14 21
6 5 9 5
2 20 2 5
8 19 2 2
6 16 28 20
1 4 17 2
1 18 23 12
