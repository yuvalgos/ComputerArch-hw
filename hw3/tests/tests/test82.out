Reading the opcodes latency info from tests/test82.dat ... Got latency for 9 opcodes
Reading the program file tests/test82.in ... Found 200 instructions
getProgDepth()==286
getInstDeps(87)=={75,76}
getDepDepth(30)==37
getDepDepth(174)==228
getDepDepth(170)==242
getDepDepth(75)==48
getInstDeps(43)=={-1,35}
getInstDeps(132)=={126,131}
getInstDeps(80)=={52,48}
getInstDeps(95)=={89,89}
getDepDepth(24)==37
getInstDeps(195)=={182,194}
getInstDeps(83)=={79,15}
getDepDepth(132)==145
getInstDeps(182)=={122,136}
getInstDeps(181)=={92,174}
getInstDeps(190)=={170,183}
getDepDepth(99)==66
getInstDeps(18)=={-1,7}
getDepDepth(59)==55
getDepDepth(120)==80
getDepDepth(151)==190
getDepDepth(81)==75
getInstDeps(133)=={-1,-1}
getInstDeps(192)=={152,170}
getDepDepth(58)==73
getDepDepth(167)==173
getDepDepth(7)==13
getDepDepth(21)==13
getDepDepth(100)==115
getInstDeps(77)=={13,52}
getInstDeps(24)=={18,-1}
getDepDepth(195)==264
getInstDeps(74)=={61,13}
getDepDepth(108)==40
getInstDeps(160)=={131,149}
getDepDepth(16)==33
getDepDepth(1)==0
getInstDeps(91)=={67,83}
getInstDeps(5)=={-1,-1}
getDepDepth(133)==0
getDepDepth(3)==0
getDepDepth(147)==150
getInstDeps(22)=={-1,-1}
getInstDeps(108)=={38,38}
getInstDeps(182)=={122,136}
getDepDepth(18)==30
getDepDepth(126)==134
getDepDepth(20)==30
getDepDepth(20)==30
getDepDepth(71)==82
getDepDepth(13)==30
getDepDepth(22)==0
getInstDeps(34)=={-1,33}
getDepDepth(96)==62
getInstDeps(197)=={170,195}
getDepDepth(172)==225
getDepDepth(41)==52
getDepDepth(179)==236
getDepDepth(180)==242
getDepDepth(93)==0
getDepDepth(57)==75
getInstDeps(65)=={31,13}
getDepDepth(76)==62
getInstDeps(102)=={101,63}
getDepDepth(56)==43
getInstDeps(13)=={3,7}
getInstDeps(118)=={61,90}
getDepDepth(104)==106
getDepDepth(77)==75
getDepDepth(149)==117
getDepDepth(122)==108
getDepDepth(14)==3
getInstDeps(22)=={-1,-1}
getDepDepth(39)==0
getInstDeps(79)=={69,70}
getInstDeps(150)=={139,141}
getInstDeps(140)=={-1,139}
getDepDepth(183)==241
getInstDeps(36)=={13,28}
